aws/basic-eks-cluster/cloudformation/eks-workers.yaml.tmpl

```diff
@@ -110,15 +110,11 @@ Parameters:
   NodeAutoScalingGroupMinSize:
     Type: Number
     Description: Minimum size of Node Group ASG.
-    Default: 1
+    Default: "{{.NodeAutoScalingGroupDesiredSize}}"
```

Comment:
This was being passed down via the XLD template already so why should this be added here?

Reply:
test it

Reply:
I tested it already, wasn't it working?